src/main/cpp/teleopcontrol/TeleopControlMap.h

```diff
@@ -240,7 +240,7 @@ robin_hood::unordered_map<TeleopControlFunctions::FUNCTION, const TeleopControlA
     {TeleopControlFunctions::ARCADE_THROTTLE, driverLJoystickY},
     // {TeleopControlFunctions::HOLONOMIC_DRIVE_STRAFE, driverLJoystickX},
     // {TeleopControlFunctions::HOLONOMIC_DRIVE_ROTATE, driverRJoystickX},
-    {TeleopControlFunctions::ARCADE_STEER, driverRJoystickY}
+    {TeleopControlFunctions::ARCADE_STEER, driverRJoystickX}
```

Comment:
should have changed the function on 242 instead of the axis on 243

Reply:
Aren't we using arcade drive?